src/pages/iou/IOUModal.js

```diff
@@ -286,7 +286,24 @@ class IOUModal extends Component {
         const amount = Math.round(this.state.amount * 100);
         const currency = this.props.iou.selectedCurrencyCode;
         const comment = this.state.comment;
+        const chatReportID = lodashGet(this.props, 'route.params.reportID', '');
+
+        // If it's paying with the wallet lets call the correct service for this
+        if (paymentMethodType === CONST.IOU.PAYMENT_TYPE.EXPENSIFY) {
+            IOU.sendMoneyWithWallet(
+                chatReportID,
+                amount,
+                currency,
+                comment,
+                this.state.participants[0].login,
```

Comment:
Does this actaully pass in an array of participants? I guess this will be refined once we would test with Web PR

Reply:
yes it should be the participant array. 
I deeply dislike this participants logic, I see it's purpose, but the constant direct access to the values of the array irks me, like @stitesExpensify mentions how do we know that `participants[0]` is the owner or not, is there more information that we do not have, namely the type of the participant, I think it's something we should have

Reply:
Posted the same thing up there reposting here so that @mountiny sees it, but what if we just made a method in `reportUtils` called `getOwner`?

Reply:
We could do that as well, but I think we should do that in a follow up issue.